framework/src/org/apache/cordova/CordovaInterfaceImpl.java

```diff
@@ -115,6 +115,10 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent intent)
             savedResult = new ActivityResultHolder(requestCode, resultCode, intent);
             if (pluginManager != null) {
                 callback = pluginManager.getPlugin(initCallbackService);
+                if(callback != null) {
+                    callback.onRestoreStateForActivityResult(savedPluginState.getBundle(callback.getServiceName()),
```

Comment:
Are you sure savedPluginState will always be non-null at this point? And that it will have the expected bundle?